packages/notebook-extension/src/index.ts

```diff
@@ -475,12 +470,14 @@ function activateNotebookHandler(
   // An object for tracking the current notebook settings.
   let editorConfig = StaticNotebook.defaultEditorConfig;
   let notebookConfig = StaticNotebook.defaultNotebookConfig;
+  // Pass URL in here.
   const factory = new NotebookWidgetFactory({
     name: FACTORY,
     fileTypes: ['notebook'],
     modelName: 'notebook',
     defaultFor: ['notebook'],
     preferKernel: true,
+    baseUrl: services.serverSettings.baseUrl,
```

Comment:
I think the notebook widget already has access to the `serviceManager,` so it shouldn't be necessary to pass in the `baseUrl` individually.